permission-portal-frontend/client/src/store/index.js

```diff
@@ -53,7 +53,7 @@ const User = types
     }
     const update = flow(function* (updates) {
       try {
-        yield db.collection('users').doc(self.email).update(updates)
+        yield db.collection('users').doc(self.email).set(updates, { merge: true })
       } catch (err) {
         console.error('Error updating users', err)
```

Comment:
the error is being swallowed here, rethrowing it or not catching it will allow the caller to detect failure.